validator/src/main.rs

```diff
@@ -588,6 +596,12 @@ pub fn main() {
     let mut validator_config = ValidatorConfig::default();
     validator_config.dev_sigverify_disabled = matches.is_present("dev_no_sigverify");
     validator_config.dev_halt_at_slot = value_t!(matches, "dev_halt_at_slot", Slot).ok();
+    if matches.is_present("hard_forks") {
+        for slot in values_t_or_exit!(matches, "hard_forks", Slot).into_iter() {
```

Comment:
I wonder this and [that](https://github.com/solana-labs/solana/pull/7801/files#diff-3125b196bc2e46166465885cd9ad66adR736) could be unified at `clap-utils`.

Reply:
The amount of duplicated code doesn't feel heavy enough to me to introduce the support burden of a more generalized thing in `clap-utils` right now.  If this code ever gets larger or we have a 3rd user then that would tip the scale for me.

Reply:
Ok!